configure.ac

```diff
@@ -32,12 +32,12 @@ AC_CACHE_SAVE
 LT_INIT([shared disable-static dlopen])
 
 dnl Work out the C++ standard which we are going to use
-AX_CXX_COMPILE_STDCXX_1Y
-AX_CXX_COMPILE_STDCXX_0X
+AX_CXX_COMPILE_STDCXX_17(noext)
```

Comment:
If you leave off the noext argument (leave it blank) I don't think it hurts anything and could potentially make some obscure hypothetical compiler that doesn't support strict conformance work. Not a big deal.

Reply:
noext is required but I will fix the macro so that it doesn't overwrite CXX